docs/run/get-backend-information.ipynb

```diff
@@ -151,7 +139,7 @@
     {
      "data": {
       "text/plain": [
-       "<IBMBackend('ibm_cairo')>"
+       "<IBMBackend('simulator_statevector')>"
```

Comment:
Womp womp. Should we set `simulator=False`?

Reply:
IMO this is ok as it comes from `least_busy`, so the code will still work when the simulators are removed.